test_postgres.py

```
@@ -1,62 +1,65 @@
-        
+
         # Fetch the result
         result = cur.fetchone()
-        
+
         # Close cursor and connection
         cur.close()
         conn.close()
-        
+
         logger.info(f"Successfully connected to PostgreSQL database: {result[0]}")
```

Ruff (0.14.3)

Logging statement uses f-string

(G004)